core/commands/name.go

```diff
@@ -53,6 +48,7 @@ Resolve the value of a reference:
 
 	Subcommands: map[string]*cmds.Command{
 		"publish": PublishCmd,
+		"upload":  UploadNameCmd,
```

Comment:
upload probably isnt the best name for this feature... maybe something like `publish-record` could be used, since its not intented as a common thing?

cc @Kubuxu @diasdavid @lgierth